asapdiscovery-modeling/asapdiscovery/modeling/protein_prep_v2.py

```diff
@@ -122,6 +123,12 @@ def prep(
                 inputs, cached_outputs = ProteinPrepperBase._gather_new_tasks(
                     complex_to_prep=inputs, cached_complexs=cached_complexs
                 )
+                if use_only_cache:
+                    if inputs:
+                        logger.warning(
+                            f"Disregarding {len(inputs)} structures which could not be found in the cache."
```

Comment:
Could be misunderstanding but isn't the message the reverse? Shouldn't it be something like 

```python
f"Disregarding inputs, {len(inputs)}, only loading structures prepped in cache."
```

Reply:
Here is the message in context:

```
2023-12-14 11:49:03,683 | root | INFO | cross_docking.py | cross_docking_workflow | Loaded 219 complexes
2023-12-14 11:49:03,685 | root | INFO | cross_docking.py | cross_docking_workflow | Prepping complexes
2023-12-14 11:49:06,685 | asapdiscovery.modeling.protein_prep_v2 | INFO | protein_prep_v2.py | prep | Loaded 218 cached structures from: /lila/data/chodera/asap-datasets/full_frag_prepped_mpro_20230122/du_cache_p_only.
2023-12-14 11:49:09,801 | asapdiscovery.modeling.protein_prep_v2 | WARNING | protein_prep_v2.py | prep | Disregarding 3 structures which could not be found in the cache.
2023-12-14 11:49:09,808 | asapdiscovery.modeling.protein_prep_v2 | INFO | protein_prep_v2.py | prep | Matched 216 cached structures which will be reused.
2023-12-14 11:49:09,811 | root | INFO | cross_docking.py | cross_docking_workflow | Prepped 216 complexes
```

Reply:
I think this makes sense to me?

Reply:
Yep makes sense in context, my confusion.